easybuild/easyconfigs/z/zarr/zarr-2.1.4-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = "PythonPackage"
+
+name = 'zarr'
+version = '2.1.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://zarr.readthedocs.io/en/stable/'
+description = """Zarr is a Python package providing an implementation of compressed, chunked, N-dimensional arrays, 
+ designed for use in parallel computing."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.13'),
+]
+
+exts_list = [
+    ('asciitree', '0.3.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/A/asciitree/'],
+        'source_tmpl': 'asciitree-%(version)s.tar.gz',
+    }),
+    ('fasteners', '0.14.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/F/fasteners/'],
+        'source_tmpl': 'fasteners-%(version)s.tar.gz',
+    }),
+    ('numcodecs', '0.5.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/N/numcodecs/'],
+        'source_tmpl': 'numcodecs-%(version)s.tar.gz',
+    }),
+
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
+}
+
+sanity_check_commands = [
+    ('python', "-c 'import zarr'"),
```

Comment:
The `import` check is already done for each extension + the parent install, so there's no need to do it again.